index.html

```diff
@@ -198,10 +198,11 @@ <h3>
           associated image.
         </p>
         <p>
-          You MUST provide a {{ImageResource/label}} unless the {{ImageResource}}'s label can be inferred from
-          an <dfn>external label</dfn> in its context. For example, the
-          {{WebAppManifest/name}} can act as the {{ImageResource/label}} for
-          any member of the {{WebAppManifest/icons}} array.
+          You MUST provide a {{ImageResource/label}} unless the
```

Comment:
The "You MUST" here is kinda problematic... RFC2119 keywords only apply to conformance classes, not human authors.  
```suggestion
          Authors are strongly encouraged to add a {{ImageResource/label}} unless the
```